frontend/package.json

```diff
@@ -12,7 +12,7 @@
     "@material-ui/icons": "^3.0.2",
     "ace-builds": "^1.4.2",
     "autosuggest-highlight": "^3.1.1",
-    "bootstrap": "^4.3.1",
+    "bootstrap": "^3.3.6",
```

Comment:
Why are we going back in version?

Reply:
We didn't use the `bootstrap` node module at all earlier; instead, we had just copied-and-pasted the bootstrap v3 styles into the imports.css file.